src/main/java/com/specialwarriors/conal/notification/domain/NotificationAgreement.java

```diff
@@ -28,9 +25,8 @@ public class NotificationAgreement {
 
     private boolean isAgree;
 
-    @ManyToOne(fetch = FetchType.EAGER)
-    @JoinColumn(name = "github_repo_id")
-    private GithubRepo githubRepo;
+    @Column(name = "github_repo_id", nullable = false)
+    private Long githubRepoId;
```

Comment:
```suggestion
    @Column(name = "github_repo_id")
    private long githubRepoId;
```
기본 타입으로 설정할 경우 굳이 래퍼 타입을 쓰고 `nullable=false`를 설정할 필요가 없습니다. 굳이 래퍼 타입을 쓰신 이유가 있다면 설명 부탁드립니다.

Reply:
> 기본 타입으로 설정할 경우 굳이 래퍼 타입을 쓰고 `nullable=false`를 설정할 필요가 없습니다. 굳이 래퍼 타입을 쓰신 이유가 있다면 설명 부탁드립니다.

몰랐습니다! 수정하겠습니다